src/utils/dates.ts

```diff
@@ -10,3 +10,13 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset: number) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+export function is24HoursLater(purchaseDate: Date): boolean {
```

Comment:
This could benefit from a more descriptive name. `is24HoursLater` suggests this will only return `true` if it's exactly 24 hours later. `longerThan24HoursFromNow` (or something like that)?